cli/tools/bench/mitata.rs

```diff
@@ -9,6 +9,34 @@
 use crate::colors;
 use std::str::FromStr;
 
+fn avg_to_iter_per_s(time: f64) -> String {
```

Comment:
Can you write some unit tests for this method? It will point out an error.

Reply:
Not sure I understand the error here - can you give me a hand?